src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/EnvelopeReference.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.model.ccd;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Represents BulkScanEnvelope complex type in CCD case definitions.
+ */
+public class EnvelopeReference {
+
+    @JsonProperty
+    public String id;
+
+    @JsonProperty
+    public String action;
```

Comment:
perhaps we could either have constant strings in `CaseAction` class above or use the enum here?